insights/parsers/smbstatus.py

```diff
@@ -50,36 +22,81 @@ def __iter__(self):
         for row in self.data:
             yield row
 
+    def parse_content(self, content):
+        new_content = get_active_lines(content, '-----------')
+        if len(content) <= 1:
+            raise ParseException("Input content is empty or there is no useful parsed data.")
+        return new_content
+
 
 @parser(Specs.smbstatus_S)
 class SmbstatusS(Statuslist):
     """
-        Class for ``smbstatus -S`` command.
+    Class ``SmbstatusS`` parses the output of the ``smbstatus -S`` command.
+
+    Sample output of this command looks like::
+
+        Service      pid     Machine       Connected at                     Encryption   Signing
+        ----------------------------------------------------------------------------------------
+        share_test   13668   10.66.208.149 Wed Sep 27 10:33:55 AM 2017 CST  -            -
+
+    The format of ``smbstatus -S`` is like table, and function `parse_fixed_table`
+    could parse it.
+
+    Examples:
+        >>> smbstatuss_info.data[0]
+        {'Signing': '-', 'Service': 'share_test', 'Encryption': '-', 'pid': '13668', 'Machine': '10.66.208.149', 'Connected_at': 'Wed Sep 27 10:33:55 AM 2017 CST'}
+        >>> smbstatuss_info.data[0]['pid']
+        '13668'
 
-        The format of ``smbstatus -S`` is like table, and function `parse_fixed_table`
-        could parse it.
+    Raises:
+        ParseException: When there is no usefull data or the input content is
+        empty,  or does contain the header line.
```

Comment:
@xiangce you need to indent the second line here and on line 91 below.

Reply:
Done, Thanks @bfahr